### PR TITLE
Added product table migration

### PR DESCRIPTION
**Pull request by carlgoshert**
I ran "php artisan migrate" on my machine to make sure it works

### CODE REVIEW

**Comment by carlgoshert** [comment]
[Add Product table](https://trello.com/c/MwcEQZ9e/24-add-product-table)

**Comment by circuitfox** [comment]
Alright. This looks good. I'd like a test to make sure the migration runs correctly though.

**Comment by circuitfox** [comment]
Also, the column names should be in `snake_case` not `camelCase`, that seems to be the convention that Laravel uses.

File: database/migrations/2017_04_07_231938_create_product_table.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class CreateProductTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('Product', function (Blueprint $table) {
```
Comment (circuitfox) [inline database/migrations/2017_04_07_231938_create_product_table.php]: The columns in camelCase here should be rewritten as snake_case. `catalogPage` becomes `catalog_page`, etc.

**Review by circuitfox — CHANGES_REQUESTED** [review]

**Comment by carlgoshert** [comment]
updated based on requested changes

File: database/migrations/2017_04_07_231938_create_product_table.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('Product');
```
Comment (circuitfox) [inline database/migrations/2017_04_07_231938_create_product_table.php]: This should be 'product'.

**Review by circuitfox — CHANGES_REQUESTED** [review]

**Review by circuitfox — APPROVED** [review]

**Pull request merged**